<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credo Control Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource/icons/icons.qrc">
    <normaloff>:/icons/cctlogo.svg</normaloff>:/icons/cctlogo.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="deviceStatusGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Device status</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="logViewerGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Log</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>CCT&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Execute</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave_settings"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Setup">
    <property name="title">
     <string>&amp;Setup</string>
    </property>
    <addaction name="actionSample_editor"/>
    <addaction name="actionGeometry_editor"/>
    <addaction name="actionEdit_configuration"/>
    <addaction name="separator"/>
    <addaction name="actionCalibration"/>
    <addaction name="separator"/>
    <addaction name="actionUser_management"/>
    <addaction name="actionProject_management"/>
   </widget>
   <widget class="QMenu" name="menu_Devices">
    <property name="title">
     <string>&amp;Devices</string>
    </property>
    <addaction name="actionX_ray_source"/>
    <addaction name="actionDetector"/>
    <addaction name="actionMotors"/>
    <addaction name="actionVacuum_gauge"/>
    <addaction name="actionTemperature_stage"/>
    <addaction name="separator"/>
    <addaction name="actionDevice_connections"/>
   </widget>
   <widget class="QMenu" name="menu_Measurement">
    <property name="title">
     <string>&amp;Measurement</string>
    </property>
    <addaction name="actionScan"/>
    <addaction name="actionSingle_exposure"/>
    <addaction name="actionTransmission"/>
    <addaction name="actionScript"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionMask_editor"/>
    <addaction name="actionView_images_and_curves"/>
    <addaction name="actionView_scans"/>
    <addaction name="actionCapillary_sizing"/>
    <addaction name="actionData_reduction"/>
    <addaction name="separator"/>
    <addaction name="actionResource_consumption"/>
    <addaction name="actionOptimize_geometry"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionCommands"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionAccounting"/>
    <addaction name="actionShutter_and_beamstop"/>
    <addaction name="actionFSN_counters"/>
    <addaction name="actionResource_usage"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Setup"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Devices"/>
   <addaction name="menu_Measurement"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSample_editor"/>
   <addaction name="actionGeometry_editor"/>
   <addaction name="actionCalibration"/>
   <addaction name="separator"/>
   <addaction name="actionX_ray_source"/>
   <addaction name="actionDetector"/>
   <addaction name="actionMotors"/>
   <addaction name="actionTemperature_stage"/>
   <addaction name="separator"/>
   <addaction name="actionScan"/>
   <addaction name="actionSingle_exposure"/>
   <addaction name="actionTransmission"/>
   <addaction name="actionScript"/>
   <addaction name="separator"/>
   <addaction name="actionMask_editor"/>
   <addaction name="actionCapillary_sizing"/>
   <addaction name="actionView_images_and_curves"/>
   <addaction name="actionView_scans"/>
  </widget>
  <action name="actionSample_editor">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/sample.svg</normaloff>:/icons/sample.svg</iconset>
   </property>
   <property name="text">
    <string>Sample editor</string>
   </property>
   <property name="toolTip">
    <string>Open the sample database editor</string>
   </property>
  </action>
  <action name="actionGeometry_editor">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/geometry.svg</normaloff>:/icons/geometry.svg</iconset>
   </property>
   <property name="text">
    <string>Geometry editor</string>
   </property>
   <property name="toolTip">
    <string>Edit geometry parameters</string>
   </property>
  </action>
  <action name="actionSave_settings">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>../../../../cpt/src/cpt/widgets/mainwindow</normaloff>../../../../cpt/src/cpt/widgets/mainwindow</iconset>
   </property>
   <property name="text">
    <string>Save settings</string>
   </property>
   <property name="toolTip">
    <string>Save the settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>../../../../cpt/src/cpt/widgets/mainwindow</normaloff>../../../../cpt/src/cpt/widgets/mainwindow</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the application</string>
   </property>
  </action>
  <action name="actionCalibration">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/calibration.svg</normaloff>:/icons/calibration.svg</iconset>
   </property>
   <property name="text">
    <string>Calibration</string>
   </property>
   <property name="toolTip">
    <string>Calibrate the beam center and the sample-to-detector distance</string>
   </property>
  </action>
  <action name="actionEdit_configuration">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/editconfig.svg</normaloff>:/icons/editconfig.svg</iconset>
   </property>
   <property name="text">
    <string>Edit configuration</string>
   </property>
   <property name="toolTip">
    <string>Configuration editor for advanced users</string>
   </property>
  </action>
  <action name="actionUser_management">
   <property name="icon">
    <iconset theme="system-users">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>User management</string>
   </property>
   <property name="toolTip">
    <string>Manage users</string>
   </property>
  </action>
  <action name="actionProject_management">
   <property name="text">
    <string>Project management</string>
   </property>
   <property name="toolTip">
    <string>Manage projects</string>
   </property>
  </action>
  <action name="actionX_ray_source">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/xraysource.svg</normaloff>:/icons/xraysource.svg</iconset>
   </property>
   <property name="text">
    <string>X-ray source</string>
   </property>
   <property name="toolTip">
    <string>Control the X-ray source</string>
   </property>
  </action>
  <action name="actionDetector">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/detector.svg</normaloff>:/icons/detector.svg</iconset>
   </property>
   <property name="text">
    <string>Detector</string>
   </property>
   <property name="toolTip">
    <string>Control the detector</string>
   </property>
  </action>
  <action name="actionMotors">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/motor.svg</normaloff>:/icons/motor.svg</iconset>
   </property>
   <property name="text">
    <string>Motors</string>
   </property>
   <property name="toolTip">
    <string>Control the motors</string>
   </property>
  </action>
  <action name="actionTemperature_stage">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/circulator.svg</normaloff>:/icons/circulator.svg</iconset>
   </property>
   <property name="text">
    <string>Temperature stage</string>
   </property>
   <property name="toolTip">
    <string>Control the temperature regulator</string>
   </property>
  </action>
  <action name="actionVacuum_gauge">
   <property name="text">
    <string>Vacuum gauge</string>
   </property>
   <property name="toolTip">
    <string>Control the vacuum gauge</string>
   </property>
  </action>
  <action name="actionDevice_connections">
   <property name="icon">
    <iconset theme="network-idle">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Device connections</string>
   </property>
   <property name="toolTip">
    <string>Connect/disconnect devices</string>
   </property>
  </action>
  <action name="actionScan">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/scan.svg</normaloff>:/icons/scan.svg</iconset>
   </property>
   <property name="text">
    <string>Scan</string>
   </property>
   <property name="toolTip">
    <string>Perform scan measurements</string>
   </property>
  </action>
  <action name="actionSingle_exposure">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/exposure.svg</normaloff>:/icons/exposure.svg</iconset>
   </property>
   <property name="text">
    <string>Single exposure</string>
   </property>
   <property name="toolTip">
    <string>Take manual exposures</string>
   </property>
  </action>
  <action name="actionTransmission">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/transmission.svg</normaloff>:/icons/transmission.svg</iconset>
   </property>
   <property name="text">
    <string>Transmission</string>
   </property>
   <property name="toolTip">
    <string>Measure transmission of several samples</string>
   </property>
  </action>
  <action name="actionScript">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/script.svg</normaloff>:/icons/script.svg</iconset>
   </property>
   <property name="text">
    <string>Script</string>
   </property>
   <property name="toolTip">
    <string>Automated measurements</string>
   </property>
  </action>
  <action name="actionMask_editor">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/flipmask.svg</normaloff>:/icons/flipmask.svg</iconset>
   </property>
   <property name="text">
    <string>Mask editor</string>
   </property>
   <property name="toolTip">
    <string>Edit mask matrices</string>
   </property>
  </action>
  <action name="actionView_images_and_curves">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/exposureviewer.svg</normaloff>:/icons/exposureviewer.svg</iconset>
   </property>
   <property name="text">
    <string>View images and curves</string>
   </property>
   <property name="toolTip">
    <string>View recorded scattering patterns and scattering curves</string>
   </property>
  </action>
  <action name="actionView_scans">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/scanviewer.svg</normaloff>:/icons/scanviewer.svg</iconset>
   </property>
   <property name="text">
    <string>View scans</string>
   </property>
   <property name="toolTip">
    <string>View scan curves</string>
   </property>
  </action>
  <action name="actionCapillary_sizing">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/capillarysizer.svg</normaloff>:/icons/capillarysizer.svg</iconset>
   </property>
   <property name="text">
    <string>Capillary sizing</string>
   </property>
   <property name="toolTip">
    <string>Determine capillary position and thickness</string>
   </property>
  </action>
  <action name="actionData_reduction">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/datareduction.svg</normaloff>:/icons/datareduction.svg</iconset>
   </property>
   <property name="text">
    <string>Data reduction</string>
   </property>
   <property name="toolTip">
    <string>Post-process data</string>
   </property>
  </action>
  <action name="actionResource_consumption">
   <property name="icon">
    <iconset resource="../../resource/icons/icons.qrc">
     <normaloff>:/icons/resourceusage.svg</normaloff>:/icons/resourceusage.svg</iconset>
   </property>
   <property name="text">
    <string>Resource consumption</string>
   </property>
   <property name="toolTip">
    <string>Analyze resource consumption and internal telemetry</string>
   </property>
  </action>
  <action name="actionOptimize_geometry">
   <property name="text">
    <string>Optimize geometry</string>
   </property>
   <property name="toolTip">
    <string>Calculate the optimum geometry</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About this program</string>
   </property>
  </action>
  <action name="actionCommands">
   <property name="icon">
    <iconset theme="help-contents">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Commands</string>
   </property>
   <property name="toolTip">
    <string>Help on the script commands</string>
   </property>
  </action>
  <action name="actionAccounting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Accounting</string>
   </property>
   <property name="toolTip">
    <string>Open accounting dockwidget</string>
   </property>
  </action>
  <action name="actionShutter_and_beamstop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shutter and beamstop</string>
   </property>
   <property name="toolTip">
    <string>Show the shutter and beamstop control dockwidget</string>
   </property>
  </action>
  <action name="actionFSN_counters">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FSN counters</string>
   </property>
   <property name="toolTip">
    <string>Show file sequence numbers</string>
   </property>
  </action>
  <action name="actionResource_usage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Resource usage</string>
   </property>
   <property name="toolTip">
    <string>Show resource usage dockwidget</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="toolTip">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resource/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
